rslib/src/notetype/fields.rs

```diff
@@ -38,6 +38,7 @@ impl NoteField {
             ord: None,
             name: name.into(),
             config: NoteFieldConfig {
+                id: Some(rand::random()),
```

Comment:
I wonder if this is sufficient. If my math is correct, there's a 0.003% chance to have an id collision within a notetype with 500 fields ([Birthday Problem](https://en.wikipedia.org/wiki/Birthday_problem)). Is there an upper bound on the number of fields?
I wanted to avoid potential future problems with JS precision, but maybe I should still use an u64 and null the first 11 bits.

Reply:
I don't think there's a hard limit, but the editor might struggle if too many are used. In any case, I think we're probably better off going with the safer option. JS precision shouldn't be an issue with protobuf-es - i64s are stored as BigInts.